.github/workflows/test-pki.yml

```diff
@@ -0,0 +1,20 @@
+name: Test PKI
+on:
+  push:
```

Comment:
I saw that you remove from push the linter
Do we want to test it out with push?

Reply:
yes, I'm in a process of rebasing and cleaning up